Examples/Scripts/Python/exatrkx.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from typing import Optional, Union
+
+import acts.examples
+import acts
+from acts import UnitConstants as u
+
+
+def addExaTrkx(
+    s: acts.examples.Sequencer,
+    trackingGeometry: acts.TrackingGeometry,
+    geometrySelection: Optional[Union[Path, str]],
+    onnxModelDir: Optional[Union[Path, str]],
+    outputDirRoot: Optional[Union[Path, str]] = None,
+) -> acts.examples.Sequencer:
+
+    # Run the particle selection
+    # The pre-selection will select truth particles satisfying provided criteria
+    # from all particles read in by particle reader for further processing. It
+    # has no impact on the truth hits themselves
+    selAlg = acts.examples.TruthSeedSelector(
+        level=acts.logging.INFO,
+        ptMin=500 * u.MeV,
+        nHitsMin=9,
+        inputParticles="particles_initial",
+        inputMeasurementParticlesMap="measurement_particle_map",
+        outputParticles="particles_seed_selected",
+    )
+    s.addAlgorithm(selAlg)
+
+    inputParticles = selAlg.config.outputParticles
+
+    # Create space points
+    spAlg = acts.examples.SpacePointMaker(
+        level=acts.logging.INFO,
+        inputSourceLinks="source_links",
+        inputMeasurements="measurements",
+        outputSpacePoints="spacepoints",
+        trackingGeometry=trackingGeometry,
+        geometrySelection=acts.examples.readJsonGeometryList(str(geometrySelection)),
+    )
+    s.addAlgorithm(spAlg)
+
+    # Setup the track finding algorithm with ExaTrkX
+    # It takes all the source links created from truth hit smearing, seeds from
+    # truth particle smearing and source link selection config
+    exaTrkxFinding = acts.examples.ExaTrkXTrackFinding(
+        inputMLModuleDir=onnxModelDir,
```

Comment:
```suggestion
        inputMLModuleDir=str(onnxModelDir),
```
to allow it to be passed as a `Path`. Alternatively, make it required as a `str` in the argument header.